.github/workflows/check-webroot.yml

```diff
@@ -0,0 +1,76 @@
+# Compact Connect - Web Frontend Checks
+
+name: Check-Webroot
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on pull requests to trunk branches involving changes to web frontend files
+  pull_request:
+    branches:
+      - main
+      - development
+    paths:
+      - webroot/**
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  CheckWebroot:
+    runs-on: ubuntu-latest
+
+    env:
+      BASE_URL: /
+      VUE_APP_DOMAIN: https://d130azwkbquf2a.cloudfront.net
```

Comment:
I'm puzzled about how this is to work. Presumably both IA and CSG will competitively publishing to this URL now?

Reply:
Yep, you got it. The actual dev environment deployment is handled for a specific repo in `webroot-deploy-development.yml`. 

This workflow file is just the "checking things" workflow, to automatically run the lint, tests, etc. It ran as part of this PR in the "All checks have passed" section below. It needs these `env` keys to exist in order to run those checks, but it's not actually deploying anywhere. The initial decision was just to have the values line up with a pre-prod environment for consistency.

Reply:
OK, got it. Thank you.